internal/providers/terraform/azure/testdata/dns_a_record_test/dns_a_record_test.golden

```diff
@@ -0,0 +1,20 @@
+
+ Name                                    Monthly Qty  Unit                  Monthly Cost 
+                                                                                         
+ azurerm_dns_a_record.first1B                                                            
+ └─ DNS queries (first 1B)                     1,000  1M queries                 $400.00 
+                                                                                         
+ azurerm_dns_a_record.over1B                                                             
+ ├─ DNS queries (over 1B)                      9,000  1M queries               $1,800.00 
+ └─ DNS queries (first 1B)                     1,000  1M queries                 $400.00 
+                                                                                         
+ azurerm_dns_a_record.withoutUsage                                                       
+ └─ DNS queries (first 1B)          Monthly cost depends on usage: $0.40 per 1M queries  
+                                                                                         
+ azurerm_dns_zone.example                                                                
+ └─ Hosted zone                                    1  months                       $0.10 
```

Comment:
Hosted zone cost should be `$0.50` by following the Azure pricing page / [calculator](https://azure.com/e/3326b578a7d24e08985000719bf3e455)